examples/kubernetes/README.md

```diff
@@ -0,0 +1,235 @@
+database name, user, and password which can be injected into your application as
+env vars.
+
+1. Create a secret with information needed to access your database:
+    ```shell
+    kubectl create secret generic <YOUR-DB-SECRET> \
+        --from-literal=db_user=<YOUR-DATABASE-USER> \
+        --from-literal=db_password=<YOUR-DATABASE-PASSWORD> \
+        --from-literal=db_name=<YOUR-DATABASE-NAME>
+    ```
+2. Next, configure your application's container to mount the secrets as env 
+   vars:
+    > [proxy_with_workload_identity.yaml](proxy_with_workload_identity.yaml#L12-L30)
+    ```yaml
+       env:
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_user
+        - name: DB_PASS
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_password
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: <YOUR-DB-SECRET>
+              key: db_name
+    ```
+3. Finally, configure your application to use these values. In the example 
+above, the values will be in the env vars `DB_USER`, `DB_PASS`, and `DB_NAME`.
+
+[ksa-secret]: https://kubernetes.io/docs/concepts/configuration/secret/
+
+## Setting up a service account
+
+The first step to running the Cloud SQL proxy in Kubernetes is creating a 
+service account to represent your application. It is recommended that you create
+a service account unique to each application, instead of using the same service
+account everywhere. This model is more secure since it allows your to limit
+permissions on a per-application basis.
+
+The service account for your application needs to meet the following criteria:
+
+1. Belong to a project with the [Cloud SQL Admin API][admin-api] enabled
+1. [Has been granted][grant-sa] the 
+   [`Cloud SQL Client` IAM role (or equivalent)][csql-roles]
+   for the project containing the instance you want to connect to
+1. If connecting using private IP, you must use a 
+   [VPC-native GKE cluster][vpc-gke], in the same VPC as your Cloud SQL instance
+
+[admin-api]: https://console.cloud.google.com/flows/enableapi?apiid=sqladmin&redirect=https://console.cloud.google.com
+[grant-sa]: https://cloud.google.com/iam/docs/granting-roles-to-service-accounts
+[csql-roles]: https://cloud.google.com/iam/docs/understanding-roles#cloud-sql-roles
+[vpc-gke]: https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips
+
+## Providing the service account to the proxy
+
+Next, you need to configure Kubernetes to provide the service account to the 
+Cloud SQL Proxy. There are two recommended ways to do this. 
+
+### Workload Identity
+
+If you are using [Google Kubernetes Engine][gke],  the preferred method is to
+use GKE's [Workload Identity][workload-id] feature. This method allows you to
+bind a [Kubernetes Service Account (KSA)][ksa] to a Google Service Account 
```

Comment:
add "(GSA)" after Google Service Account same as you did for the KSA for consistency. Even though it happens right after, it's still a cognitive blip without the parenthetical reference

Reply:
It's there, just on the next line (cause markdown):

> If you are using [Google Kubernetes Engine][gke],  the preferred method is to
> use GKE's [Workload Identity][workload-id] feature. This method allows you to
> bind a [Kubernetes Service Account (KSA)][ksa] to a Google Service Account
> (GSA). The GSA will then be accessible to applications using the matching KSA.